#1 abstract test a.1

LibbyShihh opened the pull request:
Closes #1

LibbyShihh committed:
#1 Implement A.1.1 Abstract Test for Requirement Map Operation

- Implemented abstract test A.1.1 and its integration test (IT)
- Added test URL for execution
- Defined execution order

LibbyShihh committed:
Resolve mvn build issues by updating test suite and using test count variables

- Updated test suite from Features to Maps to fix mvn build error
- Modified testng.xml
- Replaced hardcoded test counts with variables to prevent build failures due to mismatched results

LibbyShihh committed:
#1 Implement A.1.1 - Removed hard-coded collection ID restriction ("mapserver_world_map")

bpross-52n reviewed (changes requested):
I get a build error due to formatting violations. Running `mvn spring-javaformat:apply `should fix this.

LibbyShihh committed:
fix:
- Resolve formatting violations
- Use Accept header (application/json) for JSON response negotiation

Closes #1

LibbyShihh committed:
refactor:
- Support relative map URLs
- Add noOfCollections parameter to limit number of tested collections
- Set connection and read timeout to avoid hanging on unresponsive map URLs

Commit:
fix: correct noOfCollections retrieval and reformat MapOperation.java

ghobona commented:
Reviewed 2026-03-16.

@LibbyShihh Change from `http` to `https` on [this line](https://github.com/opengeospatial/ets-ogcapi-maps10/blob/%231-Abstract_test_A.1/src/main/java/org/opengis/cite/ogcapimaps10/conformance/core/MapOperation.java#L65).

For more information, see [this section](https://docs.ogc.org/is/20-058/20-058.html#_731bed37-9b84-421c-b8a8-d1f0dba93a1c).

Commit:
#1 Change from http to https
